### PR TITLE
Correct the Signer module documentation

### PR DESCRIPTION
**Pull request by junkil-park**
Conceptually, Signer is a droppable struct.

## Motivation

To correct the Signer module doc

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

cargo test

### CODE REVIEW

File: language/move-stdlib/modules/Signer.move [hunk 1]
```diff
@@ -1,10 +1,10 @@
 address 0x1 {
 module Signer {
     // Borrows the address of the signer
-    // Conceptually, you can think of the `signer` as being a resource struct wrapper arround an
+    // Conceptually, you can think of the `signer` as being a struct wrapper arround an
     // address
     // ```
-    // struct Signer has key, store { addr: address }
+    // struct Signer has copy, drop { addr: address }
```
Comment (tnowacki) [inline language/move-stdlib/modules/Signer.move:7]: Signer doesn't have copy, so this should just be drop

Comment (junkil-park) [reply language/move-stdlib/modules/Signer.move]: I see. I just fixed it.

**Review by tnowacki — APPROVED** [review]
Looks good, thanks!

**Pull request merged**